I'm trying this PR with the vendored gz binaries in noble, was stuck at yaml-cpp yesterday but saw that is already fixed (thanks!), it seems now ros_gz_image is failing:

```
lucadv@noble:~/noble_ws$ colcon build --packages-select ros_gz_image
Starting >>> ros_gz_image
--- stderr: ros_gz_image                         
CMake Error at /opt/ros/rolling/opt/gz_transport_vendor/extra_cmake/lib/cmake/gz-transport/gz-transport-config.cmake:24 (add_library):
  add_library cannot create imported target "gz-transport" because another
  target with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:20 (find_package)


CMake Error at /opt/ros/rolling/opt/gz_msgs_vendor/extra_cmake/lib/cmake/gz-msgs/gz-msgs-config.cmake:24 (add_library):
  add_library cannot create imported target "gz-msgs" because another target
  with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:23 (find_package)
```

--- a/ros_gz_bridge/src/factory.hpp
+++ b/ros_gz_bridge/src/factory.hpp
@@ -107,8 +107,7 @@ public:
   {
     std::function<void(const GZ_T &,
       const gz::transport::MessageInfo &)> subCb =
-      [this, ros_pub](const GZ_T & _msg,
-        const gz::transport::MessageInfo & _info)
+      [this, ros_pub](const GZ_T & _msg, const gz::transport::MessageInfo & _info)
       {
         // Ignore messages that are published from this bridge.
         if (!_info.IntraProcess()) {

